gltfTutorial/gltfTutorial_002_BasicGltfStructure.md

```diff
@@ -0,0 +1,127 @@
+Previous: [Introduction](gltfTutorial_001_Introduction.md) | [Table of Contents](README.md) | Next: [A Minimal glTF File](gltfTutorial_003_MinimalGltfFile.md)
+
+
+# The Basic Structure of glTF
+
+The core of glTF is a JSON file. This file describes the whole contents of the 3D scene. It consists of a description of the scene structure itself, which is given by a hierarchy of nodes that define a scene graph. The 3D objects that appear in the scene are defined using meshes that are attached to the nodes. Materials define the appearance of the objects. Animations describe how the 3D objects are transformed (e.g., rotated to translated) over time, and skins define how the geometry of the objects is deformed based on a skeleton pose. Cameras describe the view configuration for the renderer.
+
+## The JSON structure
+
+The scene objects are stored in dictionaries in the JSON file. They can be accessed using an ID, which is the key of the dictionary:
+
+```javascript
+"meshes": {
+    "FirstExampleMeshId": { ... },
+    "SecondExampleMeshId": { ... },
+    "ThirdExampleMeshId": { ... }
+}
+```
+
+
+These IDs are also used to define the *relationships* between the objects. The example above defines multiple meshes, and a node may refer to one of these meshes, using the mesh ID, to indicate that the mesh should be attached to this node:
+
+```javascript
+"nodes:" {
+    "FirstExampleNodeId": {
+        "meshes": [
+            "FirstExampleMeshId"
+        ]
+    },
+    ...
+}
+```
+
+The following image (adapted from the [glTF concepts section](https://github.com/KhronosGroup/glTF/tree/master/specification#concepts)) gives an overview of the top-level elements of a glTF:

```

Comment:
Hm ... it could be "glTF asset", but the asset is more than just the JSON. So I think I'll change this to something like "top-level elements of the JSON part of a glTF asset"